internal/collector/telemetry/process_telemetry.go

```diff
@@ -0,0 +1,105 @@
+package telemetry
+
+import (
+	"context"
+	"os"
+	"runtime"
+	"time"
+
+	"github.com/prometheus/procfs"
+	"go.opencensus.io/stats"
+	"go.opencensus.io/stats/view"
+	"go.opencensus.io/tag"
+)
+
+// ProcessMetricsViews is a struct that contains views related to process metrics (cpu, mem, etc)
+type ProcessMetricsViews struct {
+	views []*view.View
+	done  chan struct{}
+}
+
+var mRuntimeAllocMem = stats.Int64("oc.io/process/memory_alloc", "Number of bytes currently allocated in use", "By")
+var viewAllocMem = &view.View{
+	Name:        mRuntimeAllocMem.Name(),
+	Description: mRuntimeAllocMem.Description(),
+	Measure:     mRuntimeAllocMem,
+	Aggregation: view.LastValue(),
+	TagKeys:     []tag.Key{},
+}
+
+var mRuntimeTotalAllocMem = stats.Int64("oc.io/process/total_memory_alloc", "Number of bytes allocated in total", "By")
+var viewTotalAllocMem = &view.View{
+	Name:        mRuntimeTotalAllocMem.Name(),
+	Description: mRuntimeTotalAllocMem.Description(),
+	Measure:     mRuntimeTotalAllocMem,
+	Aggregation: view.Count(),
```

Comment:
is this really a count? if it is a count, total number of allocation calls, the descrption for the measure is wrong.

Reply:
fixed.